app/src/main/java/controller/customerController/CustomerSettings.java

```diff
@@ -0,0 +1,16 @@
+package controller.customerController;
+
+import java.awt.*;
+import java.io.Serializable;
+
+/**
+ * Info be used in customers creation
+ *
+ * @see CustomersController
+ */
+public class CustomerSettings implements Serializable {
```

Comment:
mozesz usunac serializable bo ja pozniej to sam dodam, na razie nie ma potrzeby. + ja nie wiem czy wysylali te dane robili przez to

Reply:
Ok